helm/ds/values.yaml

```diff
@@ -35,10 +35,10 @@ idmRepo:
   enabled: true
 
 image:
-  repository: forgerock-docker-public.bintray.io/forgerock/ds-paas
-  pullPolicy: IfNotPresent
-  #pullPolicy: Always
-  tag: 7.0.0-M134.2
+  repository: forgerock-docker-public.bintray.io/forgerock/ds
```

Comment:
DS team does not provide anymore the ds-pass archive?

Reply:
No - Matt changed it to ds

Reply:
Ok thanks for the info